### PR TITLE
[MAP] Moves the detective to above the post office

### PR DESCRIPTION
**Pull request by pudl**
:cl: Quiltyquilty
rscadd: The detective has been moved to above the post office.
/:cl:

This moves the detective's office to where the post office (the post office is now smaller) is right now, and puts a construction zone where the office is. I'm doing this because, now that the detective can be a traitor, I want to make his aesthetic/character out to be more of a barely-sec vigilante who's confined to the weird crevices of the station. This may only be the first in a series of prs to do this.

Because of this move, cargo has been downsized. Not in its main area, but the warehouse is now 4x7 (this is the same size as efficiency and dream, so it's more than functional) and the post office is smaller, but it's definitely not cramped and it still has all of the supplies as the old office. The aux tool storage has been BARELY reduced in size, and it's now the same layout as Metastation since Metastation manages to have the same supplies in less space.

Instead of leaving a thumbs down and saying it's shit, please suggest ways to improve the pr. Layout changes to the new office, different places it could go. Thanks.

New office

Old office

New post office

New cargo lobby

New warehouse

New aux tool storage

My merge token is #18588

### CODE REVIEW

**Comment by ChangelingRain** [comment]
>construction zone, a good spot for cults to set up, made smaller and put right by sec

**Comment by GunHog** [comment]
> Instead of leaving a thumbs down and saying it's shit, please suggest ways to improve the pr. Layout changes to the new office, different places it could go

Near the bridge. Add a morgue tray to the office.

**Comment by Durkel** [comment]
Need at least a second chair in the office to talk to people.

**Comment by Supermichael777** [comment]
The detective meeting people in private? Wow I thought I was suicidal sometimes.

**Comment by xxalpha** [comment]
I really don't think it should replace an existing area of the station, just expand into space.

**Comment by Xhuis** [comment]
This seems like a really odd change. The construction area is near Engineering so that engineers can build there, and the detective's office is near Security because the detective is technically part of Security. I still think that the office is in a fine spot as-is.

**Comment by Cheridan** [comment]
>vent right across from where the chair is facing so if you want to talk to the detective you have to stand on top of the vent and the chilly air blows up on you and makes you uncomfortable

shit office would not use

**Comment by optimumtact** [comment]
Why not put him in the abandoned office instead?

**Comment by Zilenan91** [comment]
I like it :+1: 

Though yeah sec should probably have access to the new construction zone so cults don't set up right outside

**Comment by Kromgar** [comment]
This is like a stealth  sec buff

**Comment by pudl** [comment]
Alright, I'm remapping the detective somewhere else. Give me a while.

**Comment by pudl** [comment]
Done.

**Comment by Cheridan** [comment]
@Kromgar uh pudl is definitely not steelpoint's alt, it's easy to see that unlike some people pudl is very cool and hip

**Comment by HankMoodyExperience** [comment]
>there are people who havent been playing long enough to remember when _all_ of security could be traitors

**Comment by Xhuis** [comment]
@HankMoodyExperience, I've been here for around three years now. I remember when Security could be traitors and I had hoped it wouldn't happen again.

**Comment by Bawhoppen** [comment]
is this where detective used to be a long time ago

**Comment by Durkel** [comment]
Still only one chair and the vent directly across the detectives chair. REEEEE

**Comment by KorPhaeron** [comment]
How does a client come sit at his desk where is the second chair

**Comment by Bawhoppen** [comment]
also yes this is retarded nice work

**Comment by as334** [comment]
Needs more windows, and then some shutters on said window, so you can look over the bleak shithole that is the station, then close the lenses and nurse a hangover.

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 1]
```diff
@@ -22358,6 +22358,12 @@
 	dir = 2;
 	icon_state = "pipe-c"
 	},
+/obj/machinery/button/door{
+	id = "datboidetective";
```
Comment (swankcookie) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: I'm all for meemees.... but can we not, tho?

Comment (optimumtact) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: it's an internal tag id, only admins will ever see it if they viewvar the door button or door

**Comment by KorPhaeron** [comment]
I still think that empty space of 4 squares looks disgusting

**Comment by PKPenguin321** [comment]
i kinda liked it when it was in the construction area because it didn't touch much else
but this is pretty bad, it basically cramps cargo right the fuck up, :-1: 
maybe put it in the vacant office?

**Comment by Zilenan91** [comment]
I like this actually. Cargo never needed all of that room in the first place.

**Comment by PKPenguin321** [comment]
oh also the old detectives office should keep its maint door

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm (hunk 1 above)
Comment (ChangelingRain) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: IIIRC you can decon buttons to change the set tag, so people might see it if they're real dedicated to checking.

**Comment by pudl** [comment]
@KorPhaeron What 4 squares?
@PKPenguin321 No map other than box had the enormous warehouse and post office. Even with it being smaller, my post office design is the most open one on the codebase. Additionally both Eff3 and Dream function perfectly fine with the adjusted warehouse size. Cargo is inexcusably big and taking space from it isn't a big deal.
@swankcookie We have meme structures.

**Comment by MrStonedOne** [comment]
>no bigger overview picture showing the new location and how it looks in relation to stuff around it

**Comment by optimumtact** [comment]
Ah lets try it, someone will revert it if it's really bad

**Comment by Jordie0608** [comment]
We created a test merging system far better suited for situations like that

**Comment by optimumtact** [comment]
Nah you guys take too long to respond

**Pull request merged**